Sources/TuistKit/Commands/Cloud/CloudProjectCreateCommand.swift

```diff
@@ -0,0 +1,33 @@
+import ArgumentParser
+import Foundation
+import TSCBasic
+import TuistSupport
+
+struct CloudProjectCreateCommand: AsyncParsableCommand {
+    static var configuration: CommandConfiguration {
+        CommandConfiguration(
+            commandName: "create",
+            _superCommandName: "project",
+            abstract: "Create a new project."
+        )
+    }
+    
+    @Argument(
+        help: "The name of the project to create.",
+        completion: .directory
+    )
+    var name: String
```

Comment:
Should we be using handle as a terminology instead of name?

Reply:
Why handle? This is a project name, by default it uses the user account. If you want to create a project for a specific organization, you need to run `tuist cloud project create project-name --organization tuist`.

Or do you think it's better for users to run `tuist cloud project create account-name/project-name`?